hack/jenkins/e2e.sh

```diff
@@ -217,6 +217,20 @@ case ${JOB_NAME} in
     NUM_MINIONS="100"
     ;;
 
+  # Runs tests on GCE soak cluster.
+  kubernetes-soak-continuous-e2e-gce)
+    : ${E2E_CLUSTER_NAME:="gce-soak-weekly"}
+    : ${E2E_NETWORK:="gce-soak-weekly"}
+    : ${E2E_ZONE:="us-central1-f"}
+    : ${GINKGO_TEST_ARGS:=" --ginkgo.skip=Skipped|network\spartition|Etcd.*SIGKILL|Density.*30\spods\
```

Comment:
Can you please introduce GCE_SOAK_CONTINUOUS_SKIP instead (and GCE_SOAK_CONTINUOUS_FLAKE if needed)?